src/keypadc/keypadc.asm

```diff
@@ -55,28 +55,24 @@ kb_ScanGroup:
 ;  arg0 : Keypad group number
 ; Returns:
 ;  Value of entire group (Keys ORd)
-	pop	hl
-	pop	bc
-	push	bc
-	push	hl
+	ld	hl,3
+	add	hl,sp
+	ld	a,(hl)
+	cp	a,8
+	jr	nc,.found
+	add	a,a
+	add	a,$0E
```

Comment:
This needs to be $10.

Reply:
Ah yes ! You're right ;)
This "ld	de,kbdG1-2" lead me to mistake!